src/Containers/Event/Event.js

```diff
@@ -2,21 +2,69 @@ import React from 'react';
 import styled from 'styled-components';
 
 const Wrapper = styled.section`
-  width: 100%;
-  border: 1px solid blue;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
+  margin: auto;
+  width: 80%;
+  padding: 2%;
+  display: flex;
+  background-color: black;
+  opacity: 80%;
+  color: white;
+  font-family: 'Raleway', sans-serif;
+  border-radius: 10px;
+
+  h3 {
+    font-size: 2.3em;
+    border-bottom: 1px solid white;
+    padding-bottom: 1%;
+  }
+
+  h4 {
+    font-size: 1.5em
+  }
+
+  h5 {
+    font-size: 1em;
+    margin: 2% auto;
+  }
 `;
 
+const InfoWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+
+  p {
+    display: inline-block;
+    font-size: .7em;
+    padding: 2%;
+    background-color: #57123A;
+    margin: 1%;
+    border-radius: 10px;
+  }
+`
+
+const ImgWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+
+  img {
+    width: 250px;
+    height: 250px;
+    margin: auto;
+  }
+`
+
 const Event = (props) => {
 
   const performers = props.artists.map((artist)=> {
-    return (
-    <p key={artist.id}>
-      {artist.name}
-    </p>
-    )
+    if (props.artists.indexOf(artist) === props.artists.length -1) {
+      return `& ${artist.name}`
+    } else {
+      return `${artist.name}, `
+    }
```

Comment:
Logic to decide what exactly to put if the performer has more performers following it, or if it is the last performer.

Reply:
Nice!